code/common/dataaccess.q

```diff
@@ -26,6 +26,7 @@ metainfo:([tablename:`$()]partfield:`$();metas:();proctype:`$());
 //-   - load config for checking input parameters
 //-   - write meta info for tables in current process to .dataaccess.metainfo
 init:{[tablepropertiespath]
+  if[.aqrest.loadexecute; .aqrest.execute::{[req;props] @[value;req;{(neg .z.w)(.gw.formatresponse[0b;0b;"error: ",x])}]}];
```

Comment:
Make the definition of .aqrest.execute single colon rather than double colon definition
Also, I think you should check for existence
```suggestion
  if[@[value;.aqrest.loadexecute;0b]; .aqrest.execute:{[req;props] @[value;req;{(neg .z.w)(.gw.formatresponse[0b;0b;"error: ",x])}]}];
```